tests/subsys/llext/simple/testcase.yaml

```diff
@@ -87,6 +84,8 @@ tests:
       - CONFIG_LLEXT_EXPORT_BUILTINS_BY_SLID=y
   llext.simple.writable_relocatable_slid_linking:
     arch_allow: arm xtensa riscv
+    platform_exclude:
+      - qemu_arc/qemu_arc_hs5x  # See #80949
```

Comment:
I think this is missing an `allow: arc`, otherwise the exclusion is irrelevant.

Reply:
Oops, added!